collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/audit/AuditTest.kt

```diff
@@ -55,4 +59,31 @@ class AuditTest {
         assertThat(auditLog[10].get("event"), equalTo("form exit"))
         assertThat(auditLog[11].get("event"), equalTo("form finalize"))
     }
+
+    @Test // https://github.com/getodk/collect/issues/5253
+    fun navigatingBackToTheFormAfterKillingTheAppWhenMovingBackwardsIsDisabled_savesFormResumeEventToAuditLog() {
+        rule.startAtMainMenu()
+            .copyForm("one-question-audit.xml")
+            .openProjectSettingsDialog()
+            .clickSettings()
+            .clickAccessControl()
+            .clickFormEntrySettings()
+            .clickMovingBackwards()
+            .clickOnString(org.odk.collect.strings.R.string.yes)
+            .pressBack(AccessControlPage())
+            .pressBack(ProjectSettingsPage())
+            .pressBack(MainMenuPage())
+            .startBlankForm("One Question Audit")
+            .fillOut(
+                FormEntryPage.QuestionAndAnswer("what is your age", "31")
+            )
+            .let {
+                CollectHelpers.killAndReopenApp()
+            }
+
+        MainMenuPage().startBlankForm("One Question Audit")
```

Comment:
I think it'd make sense for the test to save and exit the form as well to check the audit log is in the state we think it should be after the session has ended (as that's what would get uploaded).

Reply:
Fixed.